Makefile

```diff
@@ -51,6 +51,7 @@ help:  ## Display this help
 
 # Image URL to use all building/pushing image targets
 IMG ?= baremetal-operator:latest
+ACCOUNT ?= platform9
```

Comment:
Have 2 vars instead for image repo and version:
IMG_TAG=latest
IMG_REPO=platform9/baremetal-operator